### ISSUE TITLE
allow swapping to see built source

### ISSUE THREAD

**Issue by georgejecook** [rokucommunity/brighterscript#90]
much like .bs/brs/xml

I have this for my current compiler; but it's just as important for this one - many of the errors I get do not map to anything in the ide, so I have to manually find files in terminal.

I have code to do this; just need some help identifying the right file. I'll ping you about it.

This is really important imo; especially in the early phases, where some of the compile errors are the output from the compiler, not the actual source line.

**Comment by TwitchBronBron** [comment]
Yes, I agree this would be a very useful feature to have. What if we made it work like the markdown editor? There would be a command so you could assign it to a keyboard shortcut, but also it would just open the preview to the side. I think that would probably allow it to follow the debug breakpoints, or at least sync the clicks across the file and its preview. 

![markdown preview](https://user-images.githubusercontent.com/2544493/82744650-6e7bc500-9d49-11ea-997e-be5ae319aa4d.gif)

**Comment by TwitchBronBron** [comment]
I started working on this. here's what I have so far. Would that work for you?
![preview](https://user-images.githubusercontent.com/2544493/82755066-520a7780-9d9f-11ea-9e62-57b81b7613b8.gif)
